packages/api-explorer/src/components/SelectorContainer/SdkLanguageSelector.spec.tsx

```diff
@@ -60,25 +75,19 @@ describe('SdkLanguageSelector', () => {
     })
   })
 
-  test('it stores the selected language in localStorage', async () => {
```

Comment:
Deleted this test, leaving for integration testing which will be implemented within this month. Cannot have this test within the unit test since it relies on an action being dispatched in APIExplorer.tsx.